src/components/Quiz.js

```diff
@@ -72,13 +73,17 @@ const Quiz = (props) => {
 
   return (
     <div className="Quiz">
+      <Paper style={{maxHeight: 850, overflow: 'auto'}}>
+
+
       <ul list-style-type="none">
         {questionList.map((item, id) => (
           <li key={id}>
             <Question questionName={item} update={handleUpdate} />
           </li>
```

Comment:
Is it possible to get ride of the list-style? Right now its showing bullet points which doesn't really look clean necessarily. 
If not, just change the li tag to ol for numbered list.